Sources/LanguageServerProtocol/FoldingRange.swift

```diff
@@ -0,0 +1,51 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2018 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+/// A single folding range result.
+public struct FoldingRange: ResponseType, Hashable {
+
+  /// The zero-based line number from where the folded range starts.
+  public var startLine: Int
+
+  /// The zero-based character offset from where the folded range starts.
+  /// If not defined, defaults to the length of the start line.
+  public var utf16StartIndex: Int?
```

Comment:
Should be `startUTF16Index` and `endUTF16Index` so that the connection with `startLine` and `endLine` is clearer.  I think I told you the wrong thing when I suggested you rename these, sorry about that!